### ISSUE TITLE
Build installer

### ISSUE THREAD

**Issue by chris-deep** [tryphotino/photino.Native#129]
I've searched a little and am not sure exactly how to build a Mac and Win (both X64 and ARM platforms for each OS) installer for my Photino app....  Where should I look thanks!

**Comment by philippjbauer** [comment]
The guide for publishing on all platforms is available as a sample in the Photino.Samples repository as a separate project. It includes a publish script and packaging templates that can be adapted for your specific situation.

Currently only available in the debug branch (as of 03/19/2024), lets find the rough edges together! :)
https://github.com/tryphotino/photino.Samples/tree/debug/Photino.PublishPhotino

**Comment by chris-deep** [comment]
Interesting.  What's odd is this is almost the exact set of steps I did for my desktop (electron with a .Net core backend on the desktop) app.  I did the dotnet publish, etc and it seemed to work ok.  So, this should be straightforward for me to test.

I've since dropped my .Net Core backend and switched to Node-only for my desktop backend but that doesn't matter in this context.  The key for is the electron replacement...  I've actually built some C++ POCs that use the WebView header library similar to probably how your photino.Native might work.  I'll check the source soon.

But, regardless, I'll start doing more testing with the framework and then give the bundle/publish scripts a go as well.  Thanks for the project!  I've tried nearly (probably all) all electron alternatives out there and I know eventually I'll move away from Electron.  I just want to find the best fit for my app.  In the end, I may just extend the photino.Native lib for my app and build my own installer.  

I'll shoot some updates when I get them done.  I'm launching on Electron over the next week so that's goal #1 at the moment.
